docker-compose.yml

```diff
@@ -12,7 +12,8 @@ services:
     environment:
       - TRUST_PROXY_HEADER=1
       - DB_URL=postgres://postgres:p0stgr3s@postgres:5432/logto
-      - ENDPOINT # Mandatory. Mapping env to the container.
+      # Mandatory. Mapping host env to the container thus GitPod can dynamically configure the pulic URL of Logto.
```

Comment:
```suggestion
      # Mandatory. Mapping host env to the container thus GitPod can dynamically configure the public URL of Logto.
```

Reply:
Btw, why mention `GitPod`? ENDPOINT is not something specific for GitPod, right?

Reply:
see [this line](https://github.com/logto-io/logto/blob/master/.gitpod.yml#L17)

Reply:
GitPod is just an example usage?

Reply:
> GitPod is just an example usage?

so does `docker-compose` - i think i can mention it also fits for dev

Reply:
Well, I'm OK then